### DISCUSSION TITLE
Table 4 - Practical 3

### DISCUSSION THREAD

**Discussion by emmamclea** [IBAHCM/RPiR#435] [Old]
Table 4 for practical 3

**Comment by ciarakg** [comment]
Ciara Goddard : ciarakg : https://github.com/IBAHCM/ciarakgSeries03.git

**Comment by emmamclea** [comment]
Hi guys, 

This is the code for my series 3 practical. 
Emma McLea: https://github.com/IBAHCM/emmamcleaSeries03.git
